web/styles/compose.css

```diff
@@ -1402,6 +1402,10 @@ textarea.new_message_textarea {
             opacity: 0.5;
         }
     }
+
+    .compose-button-disabled {
+        pointer-events: none;
+    }
```

Comment:
This looks like a functional change, but probably actually isn't? Generally call out things like this in the commit message -- I'm assuming that `disable-btn` had that style, but is used elsewhere so you're not removing it in this commit.

Reply:
Indeed the next commit shows that. I'd probably have just done both changes with one commit, and labeled it as something about replacing `disable-btn`.